examples/deck/example.js

```diff
@@ -179,21 +188,22 @@ export default class Example extends Component<
       autoHighlight: true,
 
       // Editing callbacks
-      onEdit: ({ data }) => {
+      onEdit: ({ data, editType, featureIndex, positionIndexes, position }) => {
```

Comment:
Where is the flowtype definition?

Reply:
Was planning to add flow coverage in a future PR. There's not much there yet.

Reply:
OK